drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/ListGroupItemPresenterTest.java

```diff
@@ -54,8 +56,8 @@
     @Mock
     private DivElement divElementMock;
 
-    @Mock
-    private FieldItemPresenter fieldItemPresenterMock;
+    @Spy
+    private FieldItemPresenter fieldItemPresenterSpy;
```

Comment:
@jomarko `@Spy` must be initialized, this is the reason of failing tests. (NPE is thown)

Reply:
Done